src/Page/AbstractPage.php

```diff
@@ -1202,7 +1206,27 @@ final public function hashCode()
     /**
      * Returns an array representation of the page
      *
-     * @return array  associative array containing all page properties
+     * @see ResourceInterface
+     *
+     * @return array
+     * @psalm-return array{
```

Comment:
If this was set as a custom type here, it could then be reused and extended in child classes. What do you think?

Reply:
Can't really use + extend it in child classes (at least not with array shapes :D ) 

Will need to stay as-is for now.